pybloom_live/pybloom.py

```diff
@@ -291,6 +291,43 @@ def add(self, key):
         filter.add(key, skip_check=True)
         return False
 
+    def copy(self):
+        """Return a copy of this scalable bloom filter.
+        """
+        new_filter = ScalableBloomFilter(initial_capacity=self.initial_capacity,
+                                         error_rate=self.error_rate,
+                                         mode=self.SMALL_SET_GROWTH)
```

Comment:
This is to assume `self.SMALL_SET_GROWTH` is used all the time. Why do we need to make such an assumption? Would `self.scale`be better for this purpose? Or perhaps using [`deepcopy`](https://docs.python.org/2/library/copy.html#copy.deepcopy) is ever better, so this instance method will not look like a class method?